files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -122,6 +131,11 @@ public function register() {
 	 */
 	public function stream_open( $path, $mode, $options, &$opened_path ) {
 		$path = $this->trim_path( $path );
+		$mode = rtrim( $mode, 'bt' );
```

Comment:
What does the `rtrim()` for `bt` do?

Reply:
`b` is force binary mode and `t` is Windows-only text mode. It's used along with the regular modes of `fopen()` like so: `rb` or `r+b`. Both of which does not apply, I believe, to the functionality of the stream wrapper thus we're stripping them here.